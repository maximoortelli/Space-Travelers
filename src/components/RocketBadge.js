import styles from '../styles/rocketBadge.module.css';

function RocketBadge() {
  return <span className={`${styles.rocketBadge} ${styles.active}`}>Reserved</span>;
}

export default RocketBadge;
