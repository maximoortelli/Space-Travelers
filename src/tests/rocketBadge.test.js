import renderer from 'react-test-renderer';
import RocketBadge from '../components/RocketBadge';

test('Checks if the badge renders correctly', () => {
  const component = renderer.create(<RocketBadge />);
  const badgeTree = component.toJSON();
  expect(badgeTree).toMatchSnapshot();
});
