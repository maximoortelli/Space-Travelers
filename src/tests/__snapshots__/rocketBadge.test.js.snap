// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checks if the badge renders correctly 1`] = `
<span
  className="rocketBadge active"
>
  Reserved
</span>
`;
